.jenkins/actions/run_standalone.sh

```diff
@@ -78,6 +78,12 @@ if [ "${SAVE_CACHE}" == "true" ] ; then
     TIMESTEPS=2
 fi
 
+# GTC backend name fix: passed as gtc_gt_* but their real name are gtc:gt:*
+if [[ $backend = gtc_gt_* ]] ; then
+    # sed explained: replace _ with :, two times
+    backend=`echo $backend | sed 's/_/:/;s/_/:/'`
```

Comment:
is this not a bit risky?
why not 
```suggestion
    backend=`echo $backend | sed 's/_/:/g'`
```

Reply:
Because in `gtc:gt:cpu_ifirst` we want to conserve the second `_`.
I'd rather be very restrictive and fail if we hit a pattern we don't know